docs/en/stack/security/authentication/kerberos-realm.asciidoc

```diff
@@ -0,0 +1,63 @@
+[role="xpack"]
+[[kerberos-realm]]
+=== Kerberos authentication
+
+You can configure {security} to support Kerberos V5 authentication,
+an industry standard protocol to authenticate users in {es}.
+
+NOTE: You cannot use the Kerberos realm to authenticate users in {kib}
+and on the transport network layer.
+
+To authenticate users with Kerberos, you need to
+{ref}/configuring-kerberos-realm.html[configure a Kerberos realm] and
+<<mapping-roles, map users to {security} roles>>.
+For more information on realm settings, see
+{ref}/security-settings.html#ref-kerberos-settings[Kerberos realm settings].
+
+[[kerberos-terms]]
+==== Key concepts
+
```

Comment:
I added this content here instead of https://github.com/elastic/elasticsearch/pull/32662, since it seems like a better fit with the overview info. If you disagree, let me know!